autoload/fzf/OmniSharp.vim

```diff
@@ -24,10 +24,11 @@ function! fzf#OmniSharp#FindSymbols(quickfixes) abort
   for quickfix in s:quickfixes
     call add(symbols, s:format_line(quickfix))
   endfor
-  call fzf#run({
+  let fzf_options = len(g:OmniSharp_fzf_options) ? g:OmniSharp_fzf_options : { 'down': '40%' }
+  call fzf#run(
+  \ extend(fzf_options, {
```

Comment:
I opted to use `fzf_options` first in the `extend` to:
- avoid accidental overriding of important keys like `source` and `sink`
- `extend` doesn't look like it merges nested dictionaries so if it were second it wouldn't work for a config like `let g:OmniSharp_fzf_options = { 'window': { 'width': 0.9, 'height': 0.9 } }`